src/Sulu/Component/Rest/ListBuilder/Doctrine/FieldDescriptor/DoctrineGroupConcatFieldDescriptor.php

```diff
@@ -61,9 +71,15 @@ public function getSelect()
     }
 
     /**
-     * Returns all the joins required for this field.
-     *
-     * @return DoctrineJoinDescriptor[]
+     * {@inheritdoc}
+     */
+    public function getSimpleSelect()
```

Comment:
returns a simple select statement for where clauses (for example group-concat returns simple the field select here.